dht-node/.env.dist

```diff
@@ -1,3 +1,5 @@
+CONFIG_VERSION=v2.2023-02-03
```

Comment:
please not have the config version in here, we removed that

Reply:
deleted